docs/content/3.manual/manual.yml

```diff
@@ -408,15 +412,16 @@ sections:
           Like `.[]`, but no errors will be output if . is not an array
           or object.
 
-      - title: "`,`"
+      - title: "Comma, Output Stream Join: `,`"
```

Comment:
(Avoiding surprising new users with generators this early is partly why I'd like to call this just `Comma`.  There's a whole section later about generators.)